test/test_tsi_data_spline.jl

```diff
@@ -0,0 +1,29 @@
+import Plots
+import Dates
+
+tsi_data_spline = TSIDataSpline(FT)
+monthly_dates, tsi_data = Insolation._get_tsi_data()
+
+# Evaluate exactly at each date
+for (date, tsi_val) in zip(monthly_dates, tsi_data)
+    @test evaluate(tsi_data_spline, date) ≈ tsi_val
+end
+
+dates = collect(first(monthly_dates):Dates.Day(1):last(monthly_dates))
+tsi = evaluate.(tsi_data_spline, dates)
+tsi_data_spline_plot = Plots.plot(dates, tsi)
+Plots.savefig(tsi_data_spline_plot, "tsi_data_spline.png")
+
+# Solar cycle 24
+# Approximate dates for solar min and max
+solar_min_date = Dates.DateTime("2008-12-1")
+solar_max_date = Dates.DateTime("2014-3-1")
+solar_min = evaluate(tsi_data_spline, solar_min_date)
+solar_max = evaluate(tsi_data_spline, solar_max_date)
+
+@test solar_min <= solar_max
```

Comment:
```suggestion
@test solar_max - solar_min > 1  # More than 1 W/m2 difference in TSI between solar max and min 
```
(Make this is a bit more stringent.)